jobs/assets-subsystem/asset-sender/src/main/java/com/paladincloud/common/AssetDocumentFields.java

```diff
@@ -0,0 +1,55 @@
+package com.paladincloud.common;
+
+/**
+ * These are the fields in the ElasticSearch Asset documents
+ */
+public interface AssetDocumentFields {
```

Comment:
Logical order is usually best for defining models in Java, especially for classes with many fields or complex relationships. It groups related fields together, making the model easier to read, maintain, and understand.